package.json

```diff
@@ -2,6 +2,7 @@
   "name": "symbol-observable",
   "version": "0.2.1",
   "description": "Symbol.observable ponyfill",
+  "typings": "index.d.ts",
```

Comment:
Actually, just realized, this is also not needed as TypeScript picks it up, just like Node.js picks up `index.js`.

> Try to load a file named index.d.ts located in the package folder (node_modules/foo/) - this file should contain typings for the package. - http://www.typescriptlang.org/docs/handbook/typings-for-npm-packages.html

-

But you do need to add `"index.d.ts"` to the `"files"` field so it's included in the published package.